cdap-app-templates/cdap-etl/cdap-etl-proto/pom.xml

```diff
@@ -51,6 +51,11 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>com.google.guava</groupId>
```

Comment:
I think with new interface you can drop most (if not all) of pom changes

Reply:
As discussed offline, since all of these pom changes make sense so I am keeping them although unrelated to this PR.